vms/platformvm/vm_test.go

```diff
@@ -630,7 +630,7 @@ func TestAddValidatorReject(t *testing.T) {
 	_, _, err = vm.state.GetTx(tx.ID())
 	require.ErrorIs(err, database.ErrNotFound)
 
-	_, err = vm.state.GetCurrentValidator(constants.PrimaryNetworkID, nodeID)
```

Comment:
This test is configured to run with the Cortina rule set - so I do think we should be checking the pending validator set.

Reply:
It feels it's one of the tests which should be valid across fork. So we may change it to target the latest fork and test over current validators.
Or we can do it later, I am already considering a specific PR where I update all UTs to the right fork